File: python/program/Mahasiswa.py

class Mahasiswa:
    '''
    Kelas untuk menampung data2 Mahasiswa
    terdiri dari Nama, Nim, Prodi, dan Fakultas
    Masing2 dapat diset menggunakan setter
    Masing2 dapat diambil menggunakan getter
    '''
    
    def __init__(self, id, nama, nim, prodi, fakultas):
        self.id = id
        self.nama = nama
        self.nim = nim
        self.prodi = prodi
        self.fakultas = fakultas
    
    #Setter
    def setId(self, id):
        self.id = id
    
    def setName(self, name):
        self.nama = name
    
    def setNim(self, nim):
        self.nim = nim
    
    def setProdi(self, prodi):
        self.prodi = prodi
        
    def setFac(self, fakultas):
        self.fakultas = fakultas
    
    #getter
    def getId(self):
        return self.id
    
    def getName(self):
        return self.nama
    
    def getNim(self):
        return self.nim
    
    def getProdi(self):
        return self.prodi
    
    def getFac(self):
        return self.fakultas

File: java/program/Mahasiswa.java

public class Mahasiswa {
    
    /*
     * Kelas yang berisi data2 mahasiswa.
     * Terdiri ddari atribut name, nim, prodi, fakultas
     * Masing2 atribut dapat diset dan diambil menggunakan methode
     */
    private int id;
    private String name;
    private String nim;
    private String prodi;
    private String fakultas;

    public Mahasiswa(int id, String name, String nim, String prodi, String fakultas) {
        this.id = id;
        this.name = name;
        this.nim = nim;
        this.prodi = prodi;
        this.fakultas = fakultas;
    }

    // Setter
    public void setId(int id)
    {
        this.id = id;
    }

    public void setName(String name)
    {
        this.name = name;
    }

    public void setNim(String nim)
    {
        this.nim = nim;
    }

    public void setProdi(String prodi) {
        this.prodi = prodi;
    }

    public void setFakultas(String fakultas) {
        this.fakultas = fakultas;
    }

    // Getter
    public int GetId()
    {
        return this.id;
    }

    public String getName()
    {
        return this.name;
    }

    public String getNim()
    {
        return this.nim;
    }

    public String getProdi()
    {
        return this.prodi;
    }

    public String getFakultas() {
        return this.fakultas;
    }
}
